test/e2e/syncer/fromhost/configmaps.go

```diff
@@ -76,6 +85,34 @@ var _ = ginkgo.Describe("ConfigMaps are synced to host and can be used in Pods",
 				"VAL4": "defg",
 			},
 		}
+		configMap5 = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      cm5Name,
+				Namespace: cm5HostNamespace,
+			},
+			Data: map[string]string{
+				"key5": "value5",
+				"key6": "value6",
+			},
+		}
+		configMap6 = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      cm6Name,
+				Namespace: cm6Namespace,
+			},
+			Data: map[string]string{
+				"key6": "value6",
+			},
+		}
+		configMap7 = &corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      cm7Name,
+				Namespace: cm7Namespace,
+			},
+			Data: map[string]string{
+				"key7": "value7",
+			},
+		}
 
 	})
 
```

Comment:
could you delete configmaps and namespaces (except `vcluster` namespace) from the host in `ginkgo.AfterAll()`, similar to what we do in lines 120-134 in this file?